chapters/data-collection.tex

```diff
@@ -3,57 +3,58 @@
 \begin{fullwidth}
 Much of the recent push toward credibility in the social sciences has focused on analytical practices.
 However, credible development research often depends, first and foremost, on the quality of the raw data.
-This is because, when you are collecting the data yourself,
-or it is provided only to you through a unique partnership,
-there is no way for others to validate that it accurately reflects the reality
+When you are using original data - whether collected for the first time through surveys or sensors or acquired through a unique partnership -
+there is no way for others to validate that it accurately reflects reality
 and that the indicators you have based your analysis on are meaningful.
 This chapter details the necessary components for a high-quality data acquisition process,
-no matter whether you are recieving large amounts of unique data from partners
-or fielding a small, specialized custom survey.
-It begins with a discussion of some key ethical and legal descriptions
-to ensure that you have the right to do research using a specific dataset.
-Particularly when sensitive data is being collected by you
-or shared with you from a program implementer, government, or other partner,
-you need to make sure these permissions are correctly granted and documented,
-so that the ownership and licensing of all information is established
-and the privacy rights of the people it describes are respected.
+no matter whether you are receiving large amounts of unique data from partners
+or fielding a small, specialized custom survey. By following these guidelines, you will be able to move on to data analysis,
+assured that your data has been obtained at high standards of both quality and security.
 
+The chapter begins with a discussion of some key ethical and legal descriptions
+to ensure that you have the right to do research using a specific dataset.
+Particularly when confidential data is being collected at your behest
```

Comment:
**behest**

```suggestion
Particularly when confidential data is collected by you and your team
```

Reply:
behest really is the right word here ...